composer.json

```diff
@@ -43,5 +44,11 @@
     },
     "support": {
         "source": "https://github.com/cakephp/upgrade"
+    },
+    "require-dev": {
+        "cakephp/cakephp": "5.x-dev",
+        "cakephp/cakephp-codesniffer": "^5.0",
+        "mikey179/vfsstream": "^1.6.8",
+        "phpunit/phpunit": "^9.5"
```

Comment:
In the past we ran into conflicts with user-land code and other rector tasks because of require-dev. 1ec3f23c  is the commit that I did back then. I wonder if the change in direction that rector took has made this a non-issue now?

Reply:
ah, i see... so that why that wasn't part of the composer.json
Then I will leave it like it is